include/constants/move_id.h

```diff
@@ -402,11 +406,14 @@
 #define MOVE_TRAPPER 0x191
 #define MOVE_POSSESS 0x192
 #define MOVE_ITEMIZE 0x193
+#define MOVE_MD1 0x194 // Unused
+#define MOVE_PROJECTILE 0x195
 #define MOVE_HURL 0x196
 #define MOVE_MOBILE 0x197
 #define MOVE_ITEM_TOSS 0x198
 #define MOVE_SEE_STAIRS 0x199
 #define MOVE_LONG_TOSS 0x19A
+#define MOVE_MD1_2 0x19B // Unused
```

Comment:
Whats the `MD1` Prefix mean here?

Reply:
https://github.com/pret/pmd-red/pull/60/files#diff-059e3ff7eb81a0797b026f1fd75cf0bb9cdf5c956b1b0e0975a75b7ffd0fe326R8293

Reply:
:+1: I see